src/kata6.joinNames.js

```diff
@@ -1,3 +1,7 @@
-const joinNames = (namesObj) => {};
-
+const joinNames = (namesObj) => {
+    const namesArray = namesObj.map(obj=>obj.name)
+    const lastName = namesArray.pop()
+    return namesObj.length>1? `${namesArray.join(', ')} & ${lastName}`:namesObj[0].name
+};
```

Comment:
Just formatting here. As I mentioned in your last review please set up your IDE to format your code on save, it really will save you loads of time and effort
```
const joinNames = (namesObj) => {
    const namesArray = namesObj.map((obj) => obj.name);
    const lastName = namesArray.pop();
    return namesObj.length > 1
        ? `${namesArray.join(", ")} & ${lastName}`
        : namesObj[0].name;
};
```